File: src/Business/Services/AccountGroupService.java
package Business.Services;

import Business.Services.Base.*;
import Common.DataAccess.*;
import Common.Models.*;
import Common.Services.*;

public class AccountGroupService extends ReferenceParentEntityService<AccountGroup, AccountSubGroup> implements IAccountGroupService {
    public AccountGroupService(
            IGlobalDataAccess globalDataAccess,
            IAccountGroupDataAccess entityDataAccess,
            IAccountSubGroupDataAccess childEntityDataAccess) {
        super(globalDataAccess, entityDataAccess, childEntityDataAccess);
    }
}

File: backend/Business.Impl/Services/AccountGroupService.cs
using Business.Contracts.Services;
using Business.Impl.Services.Base;
using Business.Models.Entities;
using DataAccess.Contracts;
using Shared.Contracts;

namespace Business.Impl.Services;

internal sealed class AccountGroupService : GroupService<AccountGroup, Account>, IAccountGroupService
{
	public AccountGroupService(ISharedContext sharedContext, IAppUnitOfWork unitOfWork)
		: base(sharedContext, unitOfWork, unitOfWork.AccountGroupRepo, unitOfWork.AccountRepo)
	{
	}
}
